### PR TITLE
Add wranglerVersion options

### PR DESCRIPTION
**Pull request by perrornet**
There is an issue with the cloudflare/next-on-pages library that causes a "Script startup exceeded CPU time limit" error after using the publish command. This issue has been fixed in https://github.com/cloudflare/next-on-pages/issues/54. However, currently only the publish command from wrangler@2.16.0 version can run properly, so I need the wranglerVersion feature.

### CODE REVIEW

**Comment by WalshyDev** [comment]
This action uses the latest Wrangler version, which right now is 2.20. Are you trying to go back to 2.16 or use the beta branch?

**Comment by perrornet** [comment]
I want to use version 2.16, so I would like to be able to specify the version of Wrangler.

File: action.yml [hunk 1]
```diff
@@ -25,6 +25,9 @@ inputs:
   workingDirectory:
     description: "The working directory in which to run Wrangler"
     required: false
+  wranglerVersion:
+    description: "The version of Wrangler to use"
+    required: false
```
Comment (WalshyDev) [inline action.yml:30]:
```suggestion
    required: false
    default: 2
```

File: src/index.ts [hunk 2]
```diff
@@ -16,6 +16,9 @@ try {
 	const gitHubToken = getInput("gitHubToken", { required: false });
 	const branch = getInput("branch", { required: false });
 	const workingDirectory = getInput("workingDirectory", { required: false });
+	const wranglerVersion = getInput("wranglerVersion", { required: false })
+		? getInput("wranglerVersion", { required: false })
+		: "2";
```
Comment (WalshyDev) [inline src/index.ts:21]:
```suggestion
	const wranglerVersion = getInput("wranglerVersion", { required: false });
```

**Review by WalshyDev — APPROVED** [review]

**Pull request merged**